setup.py

```diff
@@ -49,6 +49,7 @@
           'pymysql==1.0.2',
           'pytest-cov==2.11.1',
           'pytest-pudb==0.7.0',
+          'python-Levenshtein==0.12.2',
```

Comment:
Notice that this library does not seem to be actively maintained. According to https://pypi.org/project/python-Levenshtein/#id7 they are looking for someone to maintain the code...you might want to consider alternatives.

Reply:
I think that request has been filled. If you go further down, they list that there is a maintainer, and mention that it has been updated in 2021 because there are active projects that depend on the library.